auditlog_security/models/auditlog_log_line.py

```diff
@@ -0,0 +1,30 @@
+# Copyright 2022-2024 Therp B.V.
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models
+from odoo.osv.expression import OR
+
+
+class AuditlogLogLine(models.Model):
+    _inherit = "auditlog.log.line"
+    _order = "create_date desc"
+
+    user_id = fields.Many2one(related="log_id.user_id")
+    method = fields.Char(related="log_id.method")
+    model_id = fields.Many2one(related="log_id.model_id", store=True)
+    res_id = fields.Integer(related="log_id.res_id")
+    allowed_group_ids = fields.Many2many(
+        "res.groups",
+        compute=lambda self: self.update({"allowed_group_ids": False}),
+        search="_search_allowed_group_ids",
+    )
+
+    def _search_allowed_group_ids(self, operator, value):
```

Comment:
@hbrunn even though this method works on shell and it returns what it is supposed to return, it does not trigger at all when some user loads the auditlog.log.line tree view. Can you speculate why?

Reply:
Note however, that records are filtered correctly on the tree view, for some user. Results are fine, the method is triggered, it works etc, though whenever I put a breakpoint and try to trigger it in the ui, nothing happens.

Reply:
you've set the rule not to be applied for read operations, see https://github.com/OCA/server-tools/pull/3029#discussion_r2071136138